Makefile

```diff
@@ -170,9 +175,10 @@ generate: ## Generate code
 .PHONY: generate-go
 generate-go: $(CONTROLLER_GEN) $(MOCKGEN) $(CONVERSION_GEN) $(KUBEBUILDER) $(KUSTOMIZE) ## Runs Go related generate targets
```

Comment:
@maelk I have added the requested changes [asked](https://github.com/metal3-io/ip-address-manager/pull/32#discussion_r630822872) in the previous PR here, PTAL, sorry I had to close the old one. `lint` and `lint-full` also has been added for /api folder. Thanks.